prime-router/metadata/schemas/STRAC/strac-covid-19.schema

```diff
@@ -5,9 +5,10 @@ topic: covid-19
 trackingElement: placer_order_id # placer_order_id is the internal STRAC id number
 basedOn: covid-19
 elements:
-  #
-  # These are the CSV fields from the STRAC file
-  #
+  - name: sender_id
+    cardinality: ONE
+    default: Strac
```

Comment:
Setting the default sender_id has been my standard way of providing this info to downstream systems - in this case, to hhsprotect.